src/server/pfs/server/driver.go

```diff
@@ -2837,6 +2837,10 @@ func (d *driver) scratchCommitPrefix(commit *pfs.Commit) string {
 }
 
 func (d *driver) checkFilePath(path string) error {
+	return checkFilePath(path)
```

Comment:
It would probably be better to just change the call site to use the function without the driver receiver.